examples/demo-app/src/constants/default-settings.js

```diff
@@ -94,7 +94,8 @@ export const LOADING_SAMPLE_LIST_ERROR_MESSAGE = 'Not able to load sample galler
 export const LOADING_SAMPLE_ERROR_MESSAGE = 'Not able to load sample';
 export const LOADING_URL_MESSAGE = 'You can use the following formats: CSV | JSON | Kepler.gl config json. Make sure the url contains the file extension.';
 export const CORS_LINK = 'https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS';
-export const KEPLER_DISCLAIMER = '* Kepler.gl will save your map data to your personal cloud storage, only people with the URL can access your map and data.';
+export const KEPLER_DISCLAIMER = '* Kepler.gl will save your map data to your personal cloud storage, only people with the URL can access your map and data. ' +
+  'You can edit/delete the data file in your Dropbox account anytime. If you do not have a Dropbox account, please signup over here';
```

Comment:
> If you do not have a Dropbox account, please signup over here

I will suggest removing this part of the sentence since right now the maps can be uploaded directly after a user has signed-up. So that we can consolidate the user flow into one single button.

Reply:
Sounds good!!